src/components/Cart/Cart.js

```diff
@@ -0,0 +1,81 @@
+import React, { useContext } from "react";
```

Comment:
Hard to follow imports. Not necessary for now, but try to maintain imports in a certain order. The following order is what I am following:
1. React-related imports
2. Chakra-related imports
3. Custom component imports
4. Custom configuration file imports

Reply:
done